code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -349,6 +349,10 @@ func (sc *StorageSmartContract) commitBlobberRead(t *transaction.Transaction,
 			"can't verify read marker: %v", err)
 	}
 
+	if err = commitRead.ReadMarker.VerifyClientID(); err != nil {
```

Comment:
Run this client id verification before the `ReadMarker.Verify` so that if it's invalid client id, we can avoid running `Verify`. Or perhaps we do thi Client verification in the `Verify()` method.